packages/preset-attributify/src/types.ts

```diff
@@ -36,4 +36,13 @@ export interface AttributifyOptions extends PresetOptions {
    * A list of attributes to be ignored from extracting.
    */
   ignoreAttributes?: string[]
+
+  /**
+   * Non-valued attributes will also match if the actual value represented in DOM is `true`.
+   * This option exists for supporting frameworks that encodes non-valued attributes as `true`.
+   * Enabling this option will break rules that ends with `true`.
+   *
+   * @default true
```

Comment:
```suggestion
   * @default false
```

I think it's better to default to false for a smaller bundle size. We could document this better to turn on when working with Solid.